tests/api_tests/nodes/test_views.py

```diff
@@ -3079,7 +3095,14 @@ def setUp(self):
         self.public_pointer = self.public_project.add_pointer(self.public_pointer_project,
                                                               auth=Auth(self.user),
                                                               save=True)
-        self.public_url = '/{}nodes/{}/node_links/{}'.format(API_BASE, self.public_project._id, self.public_pointer._id)
+        self.public_url = '/{}nodes/{}/node_links/{}/'.format(API_BASE, self.public_project._id, self.public_pointer._id)
+
+    def test_delete_node_link_no_permissions_for_target_node(self):
+        pointer_project = ProjectFactory(creator=self.user_two, is_public=False)
+        pointer = self.public_project.add_pointer(pointer_project, auth=Auth(self.user), save=True)
```

Comment:
Instead of `Auth(self.user)`, you can use `self.user.auth` (since `self.user = AuthUserFactory()`)